Tests/MongoSwiftSyncTests/ClientSessionTests.swift

```diff
@@ -44,7 +44,9 @@ final class SyncClientSessionTests: MongoSwiftTestCase {
         CollectionSessionOp(name: "aggregate") { _ = try $0.aggregate([], session: $1).next()?.get() },
         CollectionSessionOp(name: "distinct") { _ = try $0.distinct(fieldName: "x", session: $1) },
         CollectionSessionOp(name: "countDocuments") { _ = try $0.countDocuments(session: $1) },
-        CollectionSessionOp(name: "estimatedDocumentCount") { _ = try $0.estimatedDocumentCount(session: $1) }
+        CollectionSessionOp(name: "estimatedDocumentCount") { collection, _ in
```

Comment:
this should maybe be removed, but now I'm wondering how this test was passing to begin with if this op doesn't support sessions? cc @patrickfreed

Reply:
seems like a libmongoc bug that one is accepted. I tested this by passing in a session explicitly and using command monitoring to verify that session is used for the command
```
explicit session id: Optional({ "id" : { "$binary" : { "base64": "OWNGttajRqywX2fJDFdr1Q==", "subType" : "04" } } })

started(MongoSwift.CommandStartedEvent(command: { "count" : "MongoCollectionTests_testBlah", "$db" : "test", "lsid" : { "id" : { "$binary" : { "base64": "OWNGttajRqywX2fJDFdr1Q==", "subType" : "04" } } } }, databaseName: "test", commandName: "count", requestId: 10, operationId: 282475254, serverAddress: MongoSwift.Address(host: "127.0.0.1", port: 27017)))

succeeded(MongoSwift.CommandSucceededEvent(duration: 433, reply: { "n" : 2, "ok" : 1.0 }, commandName: "count", requestId: 10, operationId: 282475254, serverAddress: MongoSwift.Address(host: "127.0.0.1", port: 27017)))
```

Reply:
I opened [CDRIVER-3612](https://jira.mongodb.org/browse/CDRIVER-3612)

Reply:
So it looks like `estimatedDocumentCount` is now failing some evergreen tests. Not too sure why yet but am looking into it.

Reply:
I'm going to remove this test since it's failing after removing the session option.

Reply:
sounds good to me